libs/components/forms/src/lib/modules/checkbox/checkbox-group.component.scss

```diff
@@ -1,16 +1,80 @@
+@use 'libs/components/theme/src/lib/styles/variables' as *;
+@use 'libs/components/theme/src/lib/styles/compat-tokens-mixins' as compatMixins;
+
+@include compatMixins.sky-default-overrides('.sky-checkbox-group') {
+  --sky-override-checkbox-group-hint-text-space: #{$sky-space-xs};
+  --sky-override-checkbox-group-content-space: #{$sky-space-sm};
+  --sky-override-checkbox-group-content-with-hint-text-space: #{$sky-space-md};
+}
+
+@include compatMixins.sky-default-host-overrides() {
+  --sky-override-checkbox-group-stacked-margin-bottom-no-heading: #{$sky-space-md};
+  --sky-override-checkbox-group-stacked-margin-bottom-heading: #{$sky-space-xl};
+}
+
 :host {
   display: block;
+
+  &.sky-checkbox-stacked-heading {
+    margin-bottom: var(
+      --sky-override-checkbox-group-stacked-margin-bottom-heading,
+      var(--sky-space-gap-form-xl)
+    );
+  }
+
+  &.sky-checkbox-stacked-no-heading {
+    margin-bottom: var(
+      --sky-override-checkbox-group-stacked-margin-bottom-no-heading,
+      var(--sky-space-gap-form-l)
+    );
+  }
 }
 
-.sky-control-label {
+.sky-checkbox-group-legend {
   h3,
   h4,
   h5 {
     margin: 0;
     display: inline-block;
   }
 
-  span {
+  .sky-checkbox-group-heading-text {
     line-height: 1.1;
```

Comment:
I think this may be the wrong value for v2

Reply:
Moved this selector into the overrides. We don't want it to apply at all in v2 but need to maintain it for past themes.